lib/block.ml

```diff
@@ -141,15 +143,16 @@ let pp ?syntax ppf b =
   pp_footer ?syntax ppf ()
 
 let labels = [
-  `Label "dir"              , [`Any];
-  `Label "source-tree"      , [`Any];
-  `Label "file"             , [`Any];
-  `Label "part"             , [`Any];
-  `Label "env"              , [`Any];
-  `Label "skip"             , [`None];
-  `Label "non-deterministic", [`None; `Some "command"; `Some "output"];
-  `Label "version"          , [`Any];
-  `Prefix "set-"            , [`Any];
+  {s = "dir"              ; t = `Label ; vs = [`Any]};
```

Comment:
why did you change that type?

Reply:
It felt clearer and cleaner, however the tags could be more explicit

Reply:
Can you revert this change (and make an other PR if you really like the new record, even if I am not very fond of it..)  that'll make it easier to review.

Reply:
Okay, no problem